.github/workflows/jest-tests.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           node-version: lts/*
 
       - name: Install Yarn packages
-        run: yarn install --immutable
+        run: yarn install --immutable --ignore-scripts
```

Comment:
If we go with Prisma I'll revert this and add the `npx prisma generate` command directly in vercel

Reply:
(I mean once it will be merge so that vercel builds won't fail)